src/Core/AuthenticationHelpers/EasyAuthAuthenticationHandler.cs

```diff
@@ -21,22 +21,40 @@ namespace Azure.DataApiBuilder.Core.AuthenticationHelpers;
 /// </summary>
 public class EasyAuthAuthenticationHandler : AuthenticationHandler<EasyAuthAuthenticationOptions>
 {
+#if NET8_0_OR_GREATER
```

Comment:
where is this macro defined? NET8_0_OR_GREATER?

Reply:
Defined in the .NET8 SDK per: https://learn.microsoft.com/en-us/dotnet/standard/frameworks#preprocessor-symbols

in row for "Target Frameworks" : ".NET 5+ (and .NET Core)"